modules/vye/app/policies/vye/user_info_policy.rb

```diff
@@ -9,5 +9,11 @@ def create?
     end
 
     alias_method :show?, :create?
+
+    def access?
```

Comment:
This will need to be added to a new file. app/policies/vye_policy.rb or else the controller will not recognize it